src/NumericalAlgorithms/Interpolation/Interpolate.hpp

```diff
@@ -44,26 +47,30 @@ struct InterpolatorReceiveVolumeData;
 
 namespace intrp {
 namespace Events {
-template <size_t VolumeDim, typename Tensors, typename EventRegistrars>
+template <size_t VolumeDim, typename InterpolationTargetTag, typename Tensors,
+          typename EventRegistrars>
 class Interpolate;
 
 namespace Registrars {
-template <size_t VolumeDim, typename Tensors>
+template <size_t VolumeDim, typename InterpolationTargetTag, typename Tensors>
 struct Interpolate {
   template <typename RegistrarList>
-  using f = Events::Interpolate<VolumeDim, Tensors, RegistrarList>;
+  using f = Events::Interpolate<VolumeDim, InterpolationTargetTag, Tensors,
+                                RegistrarList>;
 };
 }  // namespace Registrars
 
-/// Does an interpolation by calling Actions on Interpolator.
-template <size_t VolumeDim, typename Tensors,
-          typename EventRegistrars =
-              tmpl::list<Registrars::Interpolate<VolumeDim, Tensors>>>
+/// Does an interpolation onto InterpolationTargetTag by calling Actions on
+/// Interpolator and InterpolationTargetTag.
```

Comment:
[nitpick] I think this would be slightly clearer if rephrased to "... on the Interpolator and InterpolationTargetTag components".